### DISCUSSION TITLE
545407E0 - Prey - Help me solve the frame skipping/frame pacing issues

### DISCUSSION THREAD

**Discussion by knerlington** [xenia-canary/game-patches#55] [Requests]
While it's a part of the original release and not an actual issue with the emulator the constant frame skipping really hurts the game and I need help fixing it. New to this and don't know how most of you go about making patches, but my plan of attack is to use Cheat engine and search for the capped framerate value of 30, select a larger bunch of addresses and change it on the fly. As a first step at least. If anyone could help me out and also tell me how you'd go about doing it I'd greatly appreciate it. Want to learn more in this department and contribute myself one day.

**Comment by bomabomaboma** [comment]
I encourage you to join the Xenia discord and ask in the _Modding_ room for assistance.
The issues section here is only for reporting problems with existing patches in this repository.
